### DISCUSSION TITLE
Option to keep empty tags

### DISCUSSION THREAD

**Discussion by KlutzyBubbles** [jesec/flood#809] [Ideas]
There are times i use tags to sort items im processing or need to do things to, but the tag often has nothing in it. When ists empty it isnt shown on the sidebar or tag dropdown selector. Some of these tags i also use for automation so trying to remember what tag to type when adding it again can be annoying.

Would be great if there was an option to keep empty tags in at least the tag selector dropdown menu.

**Comment by trim21** [comment]
some downloader, for example, qbittorrent doesn't allow you to use empty string as tag

**Comment by KlutzyBubbles** [comment]
I mean keep the tag, when it doesn't have any torrents tagged with it. An empty string tag would have no use.

For example having a tag named 'moving' but not having any torrents tagged with 'moving'

**Reply by trim21** [reply]
that's currently not very doable because flood doesn't have a tag list like qb does， it collect tag list from torrent instead

for this flood will also need to add new ui component to manage tag list

**Reply by trim21** [reply]
I do not against this feature but I also do not want to implement it, if someone send a PR I'll review it.
